### PR TITLE
Minecraft 1.19.3

### PR DESCRIPTION
**Pull request by TonimatasDEV**

### CODE REVIEW

**Comment by UeberallGebannt** [comment]
Thank you for your PR! I will take are more detailed look at it later this evening ^^

File: RECOMMEND.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ Here you can find the recommended softwares for your use case.
   - If you want to use mods on your server you can use [Forge](https://github.com/UeberallGebannt/server-softwares/blob/master/SERVERS.md#-forge) or [Fabric](https://github.com/UeberallGebannt/server-softwares/blob/master/SERVERS.md#-fabric), depending which mods you want to use.  
   
 ### 🔶 Minecraft with mods and plugins
-  - If you want to use Forge mods and plugins on **1.16.5+** you can use [Magma](https://github.com/UeberallGebannt/server-softwares/blob/master/SERVERS.md#-magma) or [Mohist](https://github.com/UeberallGebannt/server-softwares/blob/master/SERVERS.md#-mohist).
+  - If you want to use Forge mods and plugins on **1.18.2** you can use [Magma](https://github.com/UeberallGebannt/server-softwares/blob/master/SERVERS.md#-magma)
+  - If you want to use Forge mods and plugins on **1.16.5 or 1.19.2** you can use [Mohist](https://github.com/UeberallGebannt/server-softwares/blob/master/SERVERS.md#-mohist).
```
Comment (UeberallGebannt) [inline RECOMMEND.md:22]: Mohist doesn't actually support 1.19.2 yet.
I'd say we recommend Arclight for that version.
For 1.16.5 I'd say we recommend Mohist (as I don't know how well Arclight works for that version).

File: SERVERS.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Click [here](https://github.com/UeberallGebannt/server-softwares/blob/master/SER
 
 ### [✉️ Paper](https://papermc.io/)
   - Active: ✔️
-  - Version: 1.7.10-1.16.5 (Unsupported), 1.17.1-1.19.2
+  - Version: 1.7.10-1.17.1 (Unsupported), 1.18.2-1.19.3
```
Comment (UeberallGebannt) [inline SERVERS.md:45]: As 1.18.2 is listed as legacy (https://papermc.io/legacy) I think it is unsupported too.

File: SERVERS.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ Click [here](https://github.com/UeberallGebannt/server-softwares/blob/master/SER
   - Mods: ❌
 
 ### [🌬️ WindSpigot](https://github.com/Wind-Development/WindSpigot)
-  - Active: ✔️
+  - Active: ❌Temporaly (Alternatice: BurritoSpigot/SportPaper)
```
Comment (UeberallGebannt) [inline SERVERS.md:80]: You have two typos right there.
"Temporarily" and "Alternative"

File: SERVERS.md [hunk 4]
```diff
@@ -222,7 +222,7 @@ Click [here](https://github.com/UeberallGebannt/server-softwares/blob/master/SER
 
 ### [🔆 Glowstone](https://glowstone.net/)
   - Active: ✔️
-  - Version: 1.12.2, 1.16.5
+  - Version: 1.12.2, 1.16.5, 1.19 (Experimental)
```
Comment (UeberallGebannt) [inline SERVERS.md:225]: As I was unable to find a 1.16.5 dev branch, it is most likely not supported/available anymore.

File: SERVERS.md [hunk 5]
```diff
@@ -231,7 +231,7 @@ Click [here](https://github.com/UeberallGebannt/server-softwares/blob/master/SER
 
 ### [🧽 SpongeVanilla](https://www.spongepowered.org/downloads/spongevanilla/)
   - Active: ✔️
-  - Version: 1.8-1.12.2, 1.15.2-1.18.1
+  - Version: 1.8-1.19.3
```
Comment (UeberallGebannt) [inline SERVERS.md:234]: grafik

There is no version for 1.13.X and 1.14.X

File: SERVERS.md [hunk 6]
```diff
@@ -269,7 +269,7 @@ Click [here](https://github.com/UeberallGebannt/server-softwares/blob/master/SER
 
 ### [🔶 Magma](https://magmafoundation.org)
   - Active: ✔️
-  - Version: 1.12.2, 1.15.2 (Deprecated), 1.16.5, 1.18.2 (Experimental)
+  - Version: 1.12.2, 1.15.2 (Deprecated), 1.16.5 (Unsupported), 1.18.2 (Experimental)
```
Comment (UeberallGebannt) [inline SERVERS.md:272]: 1.18.2 isn't experimental anymore, it's pretty stable by now.

**Review by UeberallGebannt — CHANGES_REQUESTED** [review]
I added some comments to parts of the code, please take a look at them.
Beside from some small things, it looks very good ^^

File: SERVERS.md (hunk 6 above)
Comment (TonimatasDEV) [reply SERVERS.md:344]: I had thought about it

File: SERVERS.md (hunk 5 above)
Comment (TonimatasDEV) [reply SERVERS.md]: Done

File: SERVERS.md (hunk 4 above)
Comment (TonimatasDEV) [reply SERVERS.md]: Done

File: SERVERS.md (hunk 2 above)
Comment (TonimatasDEV) [reply SERVERS.md]: Done

File: RECOMMEND.md (hunk 1 above)
Comment (TonimatasDEV) [reply RECOMMEND.md]: Done

**Comment by TonimatasDEV** [comment]
All done

**Comment by UeberallGebannt** [comment]
Thank you for your help :)

**Pull request merged**